libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/AdaptiveDialog.cs

```diff
@@ -616,7 +771,7 @@ private void EnsureDependenciesInstalled()
                         Selector = new MostSpecificSelector { Selector = new FirstSelector() };
                     }
 
-                    Selector.Initialize(Triggers);
+                    this.Selector.Initialize(Triggers, true);
```

Comment:
>true [](start = 55, length = 4)

style nit: I like using label with boolean parameters so I have an idea what the boolean is controlling without having to lookup the definition
    Initialize(Triggers, nukehd: true) // <-- that grabs my attention..